tests/profiles-enabled/conftest.py

```diff
@@ -126,20 +127,63 @@ def user2_api_client(user2, token2):
     return client
 
 
+@pytest.fixture
+def user_alice_id():
+    return random_id()
+
+
+@pytest.fixture
+def user_bob_id():
+    return random_id()
+
+
+@pytest.fixture
+def no_user_api_client():
+    return APIClient()
+
+
+@pytest.fixture
+def user_alice(user_alice_id):
+    return passive_credentials_auth(
+        get_jwt(username='alice@shipchain.io', sub=user_alice_id, organization_id=random_id()))
+
+
+@pytest.fixture
+def user_bob(user_bob_id):
+    return passive_credentials_auth(
+        get_jwt(username='bob@shipchain.io', sub=user_bob_id, organization_id=random_id()))
+
+
+@pytest.fixture
+def client_alice(user_alice):
+    api_client = APIClient()
+    api_client.force_authenticate(user=user_alice)
+    return api_client
+
+
+@pytest.fixture
+def client_bob(user_bob):
```

Comment:
There is already a couple of users and clients defined here I don't think you need more.

Reply:
The reason I used the user_bob/alice was that it, imho, is cleaner and easier to read than "user1" and "user2" and to keep it consistent with profiles. I can change them to be user1 and user2 if that is desired.

Reply:
Then you could convert `user1` and `user2` in to `user_alice` and `user_bob`.